### ISSUE TITLE
More MVP Perks

### ISSUE THREAD

**Issue by FinchoZombie** [FinchoZombie/Rust-Projects#4]
I would like some suggestions for what you guys want me to add to this.

**Comment by TheLegend27-ttvKaaaantje** [comment]
Some better kits, more bandit and outpost tp's, more homes and a color customizer for your name in-game

**Comment by isaac61** [comment]
I agree with the color customizer, maybe some of the kits could be a new boom kit. MVPBOOM for example.

**Comment by TheLegend27-ttvKaaaantje** [comment]
nah dont do mvpboom thats gonna be way too op. Maybe get like a pvp kit or a better build kit

**Comment by isaac61** [comment]
Well PVP kit would be useless since you can just loot off someone. Better build kit is also not really needed because you already get enough with the normal kit. MVPBOOM could just give you more stuff, and since they bought MVP they should get more. Boom isn't hard to get so it should just be easier.

**Comment by TheLegend27-ttvKaaaantje** [comment]
Yes but if you spawn in a bag and you see 4 ppl infront of you and you 0 weapons in ur backpack how do u wanna loot/kill them? About the build kit ye ur right u get alot but if you pay for VIP/MVP on a sever you should expect some more resources and if boom is so easy to get there shouldnt be a kit for it ( for instance you spawn in a bag and see a 2x2 it wouldnt be really fair if you can just blow it up because you a have boom kit that other people dont have)

**Comment by FadedDay** [comment]
Maybe a smoke strike
